libraries/AP_Scripting/docs/docs.lua

```diff
@@ -2345,10 +2349,22 @@ function vehicle:get_circle_radius() end
 ---@return boolean
 function vehicle:set_target_angle_and_climbrate(roll_deg, pitch_deg, yaw_deg, climb_rate_ms, use_yaw_rate, yaw_rate_degs) end
 
+-- desc
+---@param roll_deg number
+---@param pitch_deg number
+---@param yaw_deg number
+---@param thrust number
+---@param use_yaw_rate boolean
+---@param yaw_rate_degs number
+---@return boolean
+function vehicle:set_target_angle_and_thrust(roll_deg, pitch_deg, yaw_deg, thrust, use_yaw_rate, yaw_rate_degs) end
+
+-- desc
 -- Sets the target velocity using a Vector3f object in a guided mode.
 ---@param vel_ned Vector3f_ud -- North, East, Down meters / second
+---@param align_yaw_to_target? boolean
```

Comment:
```suggestion
---@param align_yaw_to_target boolean
```
It is not optional in lua.